R/organize_data.R

```diff
@@ -66,11 +66,12 @@ organize_data.tune_results <-
     }
     preds$.hover <- format_hover(var, ...)
     app_type <- get_app_type(original_data[[y_name]])
-    new_shiny_data(preds, y_name, app_type, num_col_names, fac_col_names, x)
+    best_config <- tune::select_best(x, tune::.get_tune_metric_names(x)[1])$.config
```

Comment:
Can you note in the documentation for this function (as well as `explore()`) that the default configuration is based on the optimal value of the first metric?

Reply:
I would call it `default_config` instead.